BasicsCodelab/app/src/main/res/values/colors.xml

```diff
@@ -1,23 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
These files are not deleted if you follow the steps in the codelab. Are you sure you want to delete them from this solution code?